Cargo.toml

```diff
@@ -198,6 +198,7 @@ reqwest = { version = "0.12.9", default-features = false, features = [
 ring = "=0.17.5" # Upgrading this is possible, but a pain, so we don't want to pick up every new minor version (see: https://github.com/facebook/buck2/commit/91af40b66960d003067c3d241595fb53d1e636c8)
 rust-s3 = { version = "0.34.0-rc4", default-features = false, features = [
     "tokio-rustls-tls",
+    "fail-on-err"
```

Comment:
I'm really not sure about setting this. I was definitely getting errors locally, but the S3 client just swallows them so I had to dig to find them

Reply:
i don't really understand why we wouldn't want to get an error if there is an error. seems like an odd feature